docs/cn/clients/lite/README.md

```diff
@@ -49,12 +49,12 @@ Gaia-lite的基本设计理念遵循两个规则：
 
 原始的可信验证人集应该预先放置到其信任库中，通常这个验证人集来自 genesis 文件。 在运行时期间，如果 Gaia-lite 检测到不同的验证人集，它将验证它并将可信的验证人集保存到信任库中。
 
-![validator-set-change](../../../../clients/lite/pics/validatorSetChange.png)
+![validator-set-change](../../../kr/clients/lite/pics/validatorSetChange.png)
```

Comment:
we shouldn't have duplicated images for every language

Reply:
Yes, we won't have duplication (and there is no duplication now, I'm just linking KR images from CN translation), so the build won't fail. This issue doesn't block the deployment, so we can probably fix it after the merge.